pkg/controller/constraint/constraint_controller.go

```diff
@@ -128,6 +159,12 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 		util.SetHAStatus(instance, status)
 
 		if _, err := r.opa.AddConstraint(context.Background(), instance); err != nil {
+			r.reportTotalConstraints(
+				constraintKey,
+				Tags{
+					enforcementAction: enforcementAction,
```

Comment:
Maybe we want to classify the `unrecognized` enforcement action here?

Reply:
if it's not `deny` or `dryrun`, it'll be `unrecognized` here since it is going through validation

Reply:
ah, sweet, missed that.